package.json

```diff
@@ -1,7 +1,7 @@
 {
   "title": "activity-streams-experiment",
   "name": "activity-streams",
-  "version": "0.0.1",
```

Comment:
Maybe we should update this with `npm version`, rather than in this patch? That way we can tag/do the package.json update at the same time